### PR TITLE
feat:  Adjustment enabling Greenfield sdk Websocket as option

### PR DESCRIPTION
**Pull request by alexgao001**
### Description

`Greenfield-go-sdk` has enabled the Websocket client, this pr  is to make adjustment for it.

### Rationale

for establishing long connection with Node.

### Example

NA

### Changes

Notable changes:
* NA

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -168,10 +168,15 @@ require (
 replace (
 	cosmossdk.io/api => github.com/bnb-chain/greenfield-cosmos-sdk/api v0.0.0-20230425074444-eb5869b05fe9
 	cosmossdk.io/math => github.com/bnb-chain/greenfield-cosmos-sdk/math v0.0.0-20230425074444-eb5869b05fe9
-	github.com/bnb-chain/greenfield => github.com/bnb-chain/greenfield v0.2.3-alpha.5
-	github.com/cometbft/cometbft => github.com/bnb-chain/greenfield-cometbft v0.0.2-alpha.1
+
+	github.com/bnb-chain/greenfield => github.com/bnb-chain/greenfield v0.2.3-0.20230718035847-9c747242b339
+	github.com/bnb-chain/greenfield-go-sdk => github.com/alexgao001/greenfield-go-sdk v0.0.0-20230718041408-950cfb53e961
```
Comment (guagualvcha) [inline go.mod:173]: This may change before merge.

File: executor/greenfield_executor_test.go [hunk 2]
```diff
@@ -77,22 +76,20 @@ func TestGetConsensusStatus(t *testing.T) {
...
 func getCysString(e *GreenfieldExecutor) (string, error) {
-	height := int64(1)
-	// get init consensus state
-	validators, err := e.GetGnfdClient().TmClient.Validators(context.Background(), &height, nil, nil)
+	validators, err := e.GetGnfdClient().GetValidatorsByHeight(context.Background(), 1)
 	if err != nil {
 		return "", err
 	}
-	block, err := e.GetGnfdClient().TmClient.Block(context.Background(), &height)
+	block, err := e.GetGnfdClient().GetBlockByHeight(context.Background(), 1)
```
Comment (guagualvcha) [inline executor/greenfield_executor_test.go:83]: why block height is 1?

File: config/config.go [hunk 3]
```diff
@@ -42,6 +42,7 @@ type GreenfieldConfig struct {
 	GasLimit                  int64    `json:"gas_limit"`
 	FeeAmount                 int64    `json:"fee_amount"`
 	ChainIdString             string   `json:"chain_id_string"`
+	UseWebsocket              bool     `json:"use_websocket"`
```
Comment (guagualvcha) [inline config/config.go:45]: Please remind to change the deployment config.

**Review by guagualvcha — APPROVED** [review]

**Review by yutianwu — APPROVED** [review]

**Comment by yutianwu** [comment]
can you also update the readme since we changed the config.

File: go.mod (hunk 1 above)
Comment (alexgao001) [reply go.mod]: changed

**Pull request merged**